worker-batch-ingestion-plugin/src/main/java/com/github/cafdataprocessing/worker/ingestion/validator/ValidationFileAdapterException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019-2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafdataprocessing.worker.ingestion.validator;
+
+public class ValidationFileAdapterException extends Exception
```

Comment:
Classes should usually be `final`.

```suggestion
public final class ValidationFileAdapterException extends Exception
```